src/sentry/api/endpoints/organization_events_facets_performance.py

```diff
@@ -163,7 +164,7 @@ def get(self, request, organization):
         if tag_key in TAG_ALIASES:
             tag_key = TAG_ALIASES.get(tag_key)
 
-        def data_fn():
+        def data_fn(offset, limit):
```

Comment:
The `limit` here doesn't look used. You're passing `tag_key_limit` in both queries inside here. So unless they're always in sync, I don't think the pagination here is doing what you want.

Reply:
Yep, I was meaning to switch out the `tagKeyLimit` for just `limit` since it's the 'row' of the data, just forgot. Thanks.